src/app/data-set/data-set.component.html

```diff
@@ -0,0 +1,33 @@
+<button mat-raised-button (click)="addDataSets()" color="warn">Add projects</button>
+
+<div class="example-container mat-elevation-z8">
+    <mat-table #table [dataSource]="dataSource">
+        <ng-container matColumnDef="id">
+            <th mat-header-cell *matHeaderCellDef width="10px"> Id </th>
+            <td mat-cell *matCellDef="let element" width="10px"> {{element.id}} </td>
+        </ng-container>
+        <ng-container matColumnDef="url">
+            <th mat-header-cell *matHeaderCellDef width="60%"> URL </th>
+            <td mat-cell *matCellDef="let element" width="60%">
+                <a href="{{element.url}}">{{element.url}}</a>
+            </td>
+        </ng-container>
+        <ng-container matColumnDef="loading">
```

Comment:
I'm thinking the final column should either have the button "view instances" or the loading animation (possibly with the text "Building dataset project..." based on the status of the project.

Below the table of projects we can have a table of all instances (a different component), where annotators can explore all instances of the dataset, while the same component can be used to show only the instances of the selected project when clicking on "view project".

@simsimkic We should probably prioritize introducing the DatasetProject if you agree this makes sense, so that we can adapt the developing UI before too much work is put it by @rammba.